types/jasmine-given/index.d.ts

```diff
@@ -11,9 +12,9 @@ interface DoneFn {
   fail: (message?: Error | string) => void;
 }
 
-declare function Given(func: (done?: DoneFn) => void): void;
-declare function When(func: (done?: DoneFn) => void): void;
-declare function Then(func: (done?: DoneFn) => void): void;
-declare function Then(label: string, func: (done?: DoneFn) => void): void;
-declare function And(func: (done?: DoneFn) => void): void;
-declare function Invariant(func: (done?: DoneFn) => void): void;
+declare function Given(func: ((done: DoneFn) => void) | (() => void)): void;
```

Comment:
Just use:

```ts
declare function Given(func: (done: DoneFn) => void): void;
```

This will work:

```ts
Given(() => { 
})
```

Reply:
Indeed this would work and I was considering this.
However, this will result in intellisense suggesting (showing a tooltip with the signature of the method when you hover on `Given`) that the argument is required.
I wanted to make it explicit for documentation purposes.

@demensky What do you think?

Reply:
In typescript, in all cases, you can pass a function with fewer parameters.

Example:
You probably don’t use all `map` arguments every time you use it.

<img src="https://user-images.githubusercontent.com/10235949/58687682-017a5d00-838b-11e9-97c7-aa7c7e007714.png" width="695" height="188" />

Reply:
good point. Updated the code